website/lib/markdown/table-of-contents.js

```diff
@@ -21,17 +21,6 @@ const _ = require('lodash');
 //   'testing',
 // ];
 
-function getStructuredPageTree(pageList) {
```

Comment:
AFIACT this wasn't being used anywhere

Reply:
Yes, a leftover from some experiments I did. Since you're here, do you mind also deleting the commented code right above, too?